src/a11y-dialog.ts

```diff
@@ -156,12 +156,15 @@ export default class A11yDialog {
    * code
    */
   private fire(type: A11yDialogEvent, event?: Event) {
-    this.$el.dispatchEvent(
-      new CustomEvent(type, {
-        detail: event,
-        cancelable: true,
-      })
-    )
+    const customEvent = new CustomEvent(type, {
+      detail: event,
+      bubbles: true,
```

Comment:
I am unclear whether making all events bubbling can have unexpected side-effects, but I think it should be fine?